### ISSUE TITLE
I am getting the following error while using the repeat() function in my Workflow defenition:

### ISSUE THREAD

**Issue by lindajoseph01** [acesinc/json-data-generator#49]

**Comment by andrewserff** [comment]
Hi @lindajoseph01, I haven't verified with a test using your example, but I believe you are missing the `{}` around the item in the `config`. `config` is an array of objects.  so it should look like:
```
...
"steps":[{
        "config":[{"repeat(3)",      {
                "timestamp": "nowTimestamp()",
                "system": "BADGE",
                "actor": "bob",
                "action": "ENTER",
                "objects": ["Building 1"],
                "location": "45.5,44.3",
                "message": "Entered Building 1"
            }
         }],
...
```

Try that out and see if that works

**Comment by lindajoseph01** [comment]
Hey @andrewserff Thank you for your response:)

I tried to execute the test as you mentioned. I am getting the following error:

Error reading config: test
com.fasterxml.jackson.databind.JsonMappingException: Unexpected character (',' (code 44)): was expecting a colon to separate field name and value
 at [Source: java.io.BufferedInputStream@5b8dfcc1; line: 10, column: 38] (through reference chain: net.acesinc.data.json.generator.workflow.Workflow["steps"]->java.util.ArrayList[0]->net.acesinc.data.json.generator.workflow.WorkflowStep["config"]->java.util.ArrayList[0])
	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:210) ~[jackson-databind-2.5.2.jar:2.5.2]



I used the repeat special function exactly as described in the documentation.I want to create 3 copies of the array in config.

**Comment by andrewserff** [comment]
Can you post your full config?

**Comment by caseycrites** [comment]
Yeah, the `config` section expects a `List<Map<String, Object>>` which wouldn't support top-level arrays. 

https://github.com/everwatchsolutions/json-data-generator/blob/068e5d900ae1264dc23948e2a34475c1ec73a4ac/src/main/java/net/acesinc/data/json/generator/workflow/WorkflowStep.java#L19
